### ISSUE TITLE
Recent BN json change - freezing_point removed

### ISSUE THREAD

**Issue by ChucklesTheBeard** [NobleJake/ProjectPackRat#5]
```
DEBUG    : (json-error)
Json error: C:/[snip]/mods/ProjectSquirrel/advtech/sq_meds.json:137:22: Invalid or misplaced field name "freezing_point" in JSON data

    "quench": 15,
    "fun": 1,
    "freezing_point":
                     ^
                      -49,
    "addiction_potential": 3,
    "use_action": {


 FUNCTION : void JsonObject::report_unvisited() const
 FILE     : src/json.cpp
 LINE     : 125
 VERSION  : BN 9ebf08f
```

**Comment by Ignaramico** [comment]
in case someone is wondering, the issue still persist, but you only need to remove the "freezing point" parameter line from the file

**Comment by scarf005** [comment]
fixed by #8
